pyproject.toml

```diff
@@ -115,6 +115,7 @@ lint.isort = { known-first-party = [
 ], required-imports = [
   "from __future__ import annotations",
 ] }
+lint.pydocstyle.convention = "google"
```

Comment:
No google style, prefer the sphinx/resutructedtext one.

Reply:
@gaborbernat +1 but FYI ruff does not understand Sphinx params list. I've been disabling the rules in my projects in favor of using linters that do, for now.

Reply:
I know, but yet we use sphinx.

Reply:
Yes, I use Sphinx too. It was just unclear from your comment, so I decided to post this context.

Reply:
@gaborbernat As I assumed you are not up to reconsider the sphinx part I disabled the rule in ruff due to https://github.com/astral-sh/ruff/issues/12520

As a note, that was one of the minor reasons why we switched from sphinx to google style for docstrings, the bigger one was that the syntax was more friendly and less verbose regarding types.